canvas_account_admin_tools/requirements/base.txt

```diff
@@ -8,7 +8,7 @@ django-cached-authentication-middleware==0.2.1
 django-proxy==1.0.2
 django-redis-cache==1.7.1
 djangorestframework==3.5.3
-git+ssh://git@github.com/Harvard-University-iCommons/django-canvas-course-site-wizard.git@v1.3.10#egg=django-canvas-course-site-wizard==1.3.10
+git+ssh://git@github.com/Harvard-University-iCommons/django-canvas-course-site-wizard.git@v1.3.11#egg=django-canvas-course-site-wizard==1.3.10
```

Comment:
The version at the end needs to also be 1.3.11 for this to update, otherwise it stays at 1.3.10.